moto/rds2/models.py

```diff
@@ -1750,6 +1800,51 @@ def stop_db_cluster(self, cluster_identifier):
         cluster.status = "stopped"
         return previous_state
 
+    def start_export_task(self, kwargs):
+        export_task_id = kwargs["export_task_identifier"]
+        source_arn = kwargs["source_arn"]
+        snapshot_id = source_arn.split(":")[-1]
+        snapshot_type = source_arn.split(":")[-2]
+
+        if export_task_id in self.export_tasks:
+            raise ExportTaskAlreadyExistsError(export_task_id)
+        if snapshot_type == "snapshot" and snapshot_id not in self.database_snapshots:
+            raise DBSnapshotNotFoundError(snapshot_id)
+        elif (
+            snapshot_type == "cluster-snapshot"
+            and snapshot_id not in self.cluster_snapshots
+        ):
+            raise DBClusterSnapshotNotFoundError(snapshot_id)
+
+        if snapshot_type == "snapshot":
+            snapshot = self.database_snapshots[snapshot_id]
+        else:
+            snapshot = self.cluster_snapshots[snapshot_id]
+
+        if snapshot.status not in ["available"]:
+            raise InvalidExportSourceStateError(snapshot.status)
+
+        export_task = ExportTask(snapshot, kwargs)
+        self.export_tasks[export_task_id] = export_task
+
+        return export_task
+
+    def cancel_export_task(self, export_task_identifier):
+        if export_task_identifier in self.export_tasks:
+            export_task = self.export_tasks[export_task_identifier]
+            export_task.status = "canceled"
+            self.export_tasks[export_task_identifier] = export_task
+            return export_task
+        raise ExportTaskNotFoundError(export_task_identifier)
```

Comment:
Just want to clarify that export tasks should be unique for account and region. 
Even canceled ones persist in the list of tasks forever.